src/tests/ftest/util/server_utils.py

```diff
@@ -319,6 +319,31 @@ def stop_server(setname=None, hosts=None):
     subprocess.call(["pkill", "^ssh$"])
 
 
+def clean_server(hosts, scm=False, mountpoint="/mnt/daos"):
+    """Clean the tmpfs  on the servers.
+
+    Args:
+        hosts (list): list of host names where servers are running
+        scm (bool): if true, remove tmpfs mount
+        mountpoint (str): mount point path to clean
+    """
+    clean_cmds = [
+        "find {} -mindepth 1 -maxdepth 1 -print0 | \
+        xargs -0r rm -rf".format(mountpoint)
+    ]
+
+    if scm:
+        clean_cmds.append(
+            "sudo umount {0}; sudo rm -rf {0}".format(mountpoint))
+
+    # Intentionally ignoring the exit status of the command
+    result = pcmd(hosts, "; ".join(clean_cmds), False, None, None)
```

Comment:
You probably only want to execute subsequent commands as long as the prior ones succeed don't you?  I.e. you don't really want to run the `rm -rf` (or `rmdir` as suggested above) if the `umount` fails, do you?

So above on line 337 you probably want something more like:
```bash
"if ! sudo umount {0}; then exit 1; fi; sudo rmdir {0}".format(mountpoint))"